packages/mui-base/src/index.ts

```diff
@@ -7,12 +7,13 @@ export * from './Field';
 export * from './Fieldset';
 export * from './Form';
 export * from './Menu';
-export * from './NumberField';
+export * from './Select';
```

Comment:
Something wrong happened here

```suggestion
export * from './NumberField';
```